### PR TITLE
Feature/addLoginForm

### PR DESCRIPTION
**Pull request by byteconde**
### This pull request includes:

* Add login form
* Validate if local storage is empty
* If there's data in LS, then redirect to Tutopia channel

solved #23

### CODE REVIEW

File: public/js/login.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const $form = document.getElementById('loginForm');
+
+function getrgb() {
+  var r = Math.floor(Math.random()*256);          // Random between 0-255
+  var g = Math.floor(Math.random()*256);          // Random between 0-255
+  var b = Math.floor(Math.random()*256);          // Random between 0-255
+  var rgb = 'rgb(' + r + ',' + g + ',' + b + ')'
+  return rgb
+  }
```
Comment (condef5) [inline public/js/login.js]: Fix indentation bro.

File: public/index.html [hunk 2]
```diff
@@ -6,13 +6,22 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Tutopia Clon</title>
-  <link rel="stylesheet" href="css/style.css">
+  <link rel="stylesheet" href="css/login.css">
   <link rel="shortcut icon" href="img/favicon.png">
 </head>
 
 <body>
-  <h1>Tutopia</h1>
```
Comment (test0n3) [inline public/index.html]: Was it necessary to delete this tag?

File: public/js/login.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+const $form = document.getElementById('loginForm');
+
+function getrgb() {
+  var r = Math.floor(Math.random()*256);          // Random between 0-255
```
Comment (test0n3) [inline public/js/login.js]: Try changing `var` to `let`

File: public/js/login.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
+const $form = document.getElementById('loginForm');
+
+function getrgb() {
+  const r = Math.floor(Math.random()*256);          // Random between 0-255
+  const g = Math.floor(Math.random()*256);          // Random between 0-255
+  const b = Math.floor(Math.random()*256);          // Random between 0-255
+  const rgb = 'rgb(' + r + ',' + g + ',' + b + ')'
```
Comment (condef5) [inline public/js/login.js]: Hey you should use literals:
```
const rgb = `rgb(${r}, ${g}, ${b})`
```

File: public/js/login.js [hunk 5]
```diff
@@ -1,24 +1,24 @@
...
-  const r = Math.floor(Math.random()*256);          // Random between 0-255
-  const g = Math.floor(Math.random()*256);          // Random between 0-255
-  const b = Math.floor(Math.random()*256);          // Random between 0-255
-  const rgb = 'rgb(' + r + ',' + g + ',' + b + ')'
-  return rgb
-  }
+  const r = Math.floor(Math.random() * 256); // Random between 0-255
+  const g = Math.floor(Math.random() * 256); // Random between 0-255
+  const b = Math.floor(Math.random() * 256); // Random between 0-255
+  const rgb = `rgb(${r}, ${g}, ${b})`;
```
Comment (test0n3) [inline public/js/login.js]: Set limits, this color should not be pale or white. May be a just filter with CSS: [MDN CSS - Filter](https://developer.mozilla.org/en-US/docs/Web/CSS/filter)

File: public/js/login.js [hunk 6]
```diff
@@ -1,24 +1,24 @@
...
+  return rgb;
+}
 
-  $form.addEventListener('submit', (event) => {
```
Comment (test0n3) [inline public/js/login.js]: We really need to normalize the configuration of VS Code.

**Pull request merged**